Dockerfile.dev

```diff
@@ -1,14 +1,9 @@
-FROM ubuntu:latest
-
-RUN apt update && apt upgrade
-RUN apt install python3.10 pip -y
+FROM python:3.10.9
```

Comment:
You still change the base image to python, if everything works I'm ok with it!

Reply:
Again, I tested and it worked perfectly

Reply:
perfect!